src/ApiPort.VisualStudio/AnalyzeMenu.cs

```diff
@@ -172,8 +172,8 @@ private string[] PromptForAssemblyPaths()
             var header = new StringBuilder();
 
             header.AppendLine(string.Format(CultureInfo.CurrentCulture, LocalizedStrings.CopyrightFormat, _productInfo.Version));
-            header.AppendLine(string.Concat(CultureInfo.CurrentCulture, LocalizedStrings.AboutTool, " ", DocumentationLinks.About));
-            header.AppendLine(string.Concat(CultureInfo.CurrentCulture, LocalizedStrings.AboutPrivacy, " ", DocumentationLinks.PrivacyPolicy));
+            header.AppendLine($"{LocalizedStrings.AboutTool} {DocumentationLinks.About}");
```

Comment:
One of the issues we had from FxCop was that we weren't being clear on specifying `CultureInfo.CurrentCulture` and using `string.Format` instead. Did we disable that rule, @twsouthwick

Reply:
Nvm, It looks like the dotnet-ci builds are down.

Reply:
Interpolated strings by default are culture specific, so this should be fine. The rule you're thinking of is when we need invariant strings